Makefile

```diff
@@ -387,12 +387,12 @@ toc:
 .PHONE: markdownlint
 markdownlint:
 	@echo "===> Running markdownlint <==="
-	markdownlint -c .markdownlint-config.yml -i CHANGELOG.md -i hack/netpol -i CODE_OF_CONDUCT.md .
+	markdownlint -c .markdownlint-config.yml -i CHANGELOG/ -i CHANGELOG.md -i hack/netpol -i CODE_OF_CONDUCT.md .
```

Comment:
`-i` is for ignore
markdownlint is failing otherwise, because `CHANGELOG.md` does not have a heading, and I didn't see the point in adding one, so I'm just excluding the file

Reply:
I thought CHANGELOG.md is removed, now I see it's still there. Thanks for explanation.